jest.config.js

```diff
@@ -11,10 +11,9 @@ const customJestConfig = {
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
       '<rootDir>/jest/__mocks__/fileMock.js',
     '\\.(css|scss)$': '<rootDir>/jest/__mocks__/styleMock.js',
-    '^react(.*)$': '<rootDir>/node_modules/react$1',
   },
   testEnvironment: 'jsdom',
-  moduleDirectories: ['node_modules', '.'],
```

Comment:
Jest config modified Chakra UI's module and unit tests reported "function not found" errors. Need to fix the Jest config here.